docs/overrides/partials/tabs-item.html

```diff
@@ -1,13 +1,16 @@
 <!--
```

Comment:
What's the reason for changing this file?

Reply:
Not compatible with the way partials are overriden when using material theme 9.X.X. It threw some error with nav items. But it is a very clear change in terms of UI if it breaks, so it is easy to spot that it did not break things I would say